Cubical/HoTT-UF.agda

```diff
@@ -0,0 +1,101 @@
+{-
+
+This is a HoTT-UF core library based on cubical type theory, where the
+cubical machinery is hidden, using the HoTT Book terminology and
+notation.
+
+The point is that function extensionality, propositional truncation
+and univalence compute (an example is given below).
+
+For the moment, this requires the development version of Agda.
+
+-}
+
+{-# OPTIONS --cubical --exact-split --safe #-}
```

Comment:
What is `--exact-split` and `--safe`? Should these be added to the other files?

Reply:
* [safe](https://agda.readthedocs.io/en/v2.5.4.2/language/safe-agda.html): "By using the command-line option --safe, a user can specify that Agda should ensure that features leading to possible inconsistencies should be disabled."
* [exact split](https://agda.readthedocs.io/en/v2.5.4/tools/command-line-options.html): "Require all clauses in a definition to hold as definitional equalities ". See [case trees](https://agda.readthedocs.io/en/v2.5.4/language/function-definitions.html#case-trees). In practice you get this if you construct your terms using case split in the emacs mode.

Reply:
Thanks! I'll add `--safe` to all of the files.